CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Breaking changes
 
 - messages produced by `cmd/bnscli` have a new binary format incompatible with
   the previous version.
+- [x/gov] added indexes to proposals and electorate to enable better client-side UX
```

Comment:
Please use ` (backtick) instead of brackets. This is markdow file and we use code formatting it for cases like this.